examples/nodejs/vector-index/doc-example-files/doc-examples-nodejs-apis.ts

```diff
@@ -141,6 +142,66 @@ async function example_API_SearchAndFetchVectors(vectorClient: PreviewVectorInde
   }
 }
 
+function example_API_FilterExpressionOverview() {
+  /*
+   * For convenience, the filter expressions can be imported as follows:
+   * import { VectorFilterExpressions as F } from '@gomomento/sdk';
+   *
+   * To demonstrate the various filter expressions, suppose we have a
+   * dataset of movies with the following schema:
+   * {
+   *  movie_title: string,
+   *  year: int,
+   *  gross_revenue_millions: float,
+   *  in_theaters: bool,
+   *  actors: list<string>,
+   *  directors: list<string>,
+   * }
+   */
+
+  // Is the movie titled "The Matrix"?
+  F.equals('movie_title', 'The Matrix');
+
+  // Is the movie not titled "The Matrix"?
+  F.not(F.equals('movie_title', 'The Matrix'));
+
+  // Was the movie released in 1999?
+  F.equals('year', 1999);
+
+  // Did the movie gross 463.5 million dollars?
+  F.equals('gross_revenue_millions', 463.5);
+
+  // Was the movie in theaters?
+  F.equals('in_theaters', true);
+
+  // Was the movie not in theaters?
+  F.greaterThan('year', 1990);
+
+  // Was the movie released after 2020?
```

Comment:
Was the movie release in or after 2020?

Reply:
Fixed in https://github.com/momentohq/client-sdk-javascript/pull/1130/commits/5519040aadeaba4bcd2168243bf57988c28d0c61